cdap-cli/src/main/java/co/cask/cdap/cli/command/metadata/AddMetadataPropertiesCommand.java

```diff
@@ -34,6 +34,10 @@
 
   private final MetadataClient client;
 
+  public static final String ENTITY_DESCRIPTION_STRING = String.format("'<%s>' is of the form " +
+    "'<entity-type>:<namespace-id>.<entity-name>', where '<entity-type>' is one of " +
```

Comment:
sounds good, other than - the line about program type should be right after programs.